shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -222,7 +222,7 @@ - (nonnull NSView*)createWithViewIdentifier:(int64_t)viewId arguments:(nullable
   engine.semanticsEnabled = YES;
   EXPECT_TRUE(enabled_called);
   // Send flutter semantics updates.
-  FlutterSemanticsNode root;
+  FlutterSemanticsNode2 root;
```

Comment:
I may have missed this, why there is a FlutterSemanticsNode2? it seems to be the same as FlutterSemanticsNode?

Reply:
See [go/flutter-semantics-abi](http://go/flutter-semantics-abi) or the background section for this issue: https://github.com/flutter/flutter/issues/121176

TLDR: `FlutterSemanticsUpdate`'s ABI would break if we added new members to `FlutterSemanticNode`s or `FlutterSemanticCustomAction`s. As a result, these old types are now "frozen" and new members should be added to the new types instead. The new types were introduced by https://github.com/flutter/engine/pull/39807

Reply:
ah make sense. Thanks for explaining